packages/mdc-menu/simple/foundation.js

```diff
@@ -300,6 +301,14 @@ export default class MDCSimpleMenuFoundation extends MDCFoundation {
   }
 
   handlePossibleSelected_(evt) {
+    const targetIsDisabled =
+      this.adapter_.getAttributeForEventTarget(
+        evt.target, MDCSimpleMenuFoundation.strings.ARIA_DISABLED_ATTR
+      ) === 'true';
+    if (targetIsDisabled) {
+      evt.stopPropagation();
```

Comment:
yeah, we probably just want to `return` here. We've had lots of trouble with `stopPropagation` as it is...